### ISSUE TITLE
Missing cfi_ignorelist.txt in tuxmake images

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/continuous-integration2#503]
Recently, android-4.14 builds started failing:

https://storage.tuxsuite.com/public/clangbuiltlinux/continuous-integration2/builds/2KX6JvVGPbBd8xESslbZtC6WsUT/

```
Cannot use CONFIG_CC_STACKPROTECTOR_STRONG: -fstack-protector-strong not supported by compiler
```

This is a failed `cc-option` call with a clearly supported flag, which means that the compiler is erroring before this point. Dumping try-run calls to a log file reveals the following error:

```
clang: error: missing sanitizer ignorelist: '/usr/lib/llvm-15/lib/clang/15.0.7/share/cfi_ignorelist.txt'
```

This file was recently moved to the `libclang-rt-dev` package on Debian, which `tuxmake` does not explicitly install (since it is new) nor is `libclang-rt-dev` listed as an explicit dependency of the `clang` package (it is a "recommended" addition), hence the error.

https://salsa.debian.org/pkg-llvm-team/llvm-toolchain/-/commit/d24f4d88f13021fca6afe9ce9d087c71362af2c

I have prepared a fix for this on the `tuxmake` side by installing that package for the relevant versions:

https://gitlab.com/nathanchance/tuxmake/-/commit/8250e09a386c164fc8fbb19133d68250e86ed081

Unfortunately, the `clang-14` packages are currently broken so I cannot ask for it to be merged until that issue is cleared up.

```
The following packages have unmet dependencies:
 libclang-rt-14-dev : Breaks: libclang-common-14-dev (< 1:14.0.6-10~exp4) but 1:14.0.6~++20230111083544+f28c006a5895-1~exp1~20230111083615.217 is to be installed
E: Unable to correct problems, you have held broken packages.
```

https://github.com/llvm/llvm-project/issues/59903

**Comment by nathanchance** [comment]
cc @sylvestre, should have done that initially :)

**Comment by nathanchance** [comment]
https://gitlab.com/Linaro/tuxmake/-/merge_requests/290

**Comment by nathanchance** [comment]
This is now fixed, android-4.14 builds are currently passing.
